Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -190,6 +217,28 @@ def cybereason_api_call(
                 retries=retries,
                 backoff_factor=backoff_factor,
             )
+
+        try:
+            res = make_request()
+            # Check for redirection to login page logic
+            if res.status_code == 200 and "login" not in url_suffix and "login" in str(res.url):
+                demisto.info("cybereason_api_call: Token Trigger In Retry method: Session expired, logging in again.")
+
+                new_token, login_time = login(self)
+
+                # Update context and headers
+                integration_context = (
+                    get_integration_context()
+                )  # You might need to make integration_context global or pass it around if it's not
+                integration_context["jsession_id"] = new_token
+                integration_context["valid_until"] = login_time + 28000
+                set_integration_context(integration_context)
+                HEADERS["Cookie"] = f"JSESSIONID={new_token}"
+                demisto.info(f"cybereason_api_call: New token generated: {new_token}. Retrying original request.")
```

Comment:
This log message exposes the new session token at the `info` level. For security reasons, sensitive data such as tokens should not be logged, or if required for debugging, should only be logged at the `debug` level and preferably masked It’s recommended to use the `add_sensitive_log_strs` function for handling such cases